dev/CommonStyles/AppBarButton_rs4_themeresources.xaml

```diff
@@ -22,284 +22,7 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
```

Comment:
Will these ThemeDictionaries be merged down in a future commit or a future PR? I would have expected to see only one file per control at this point.

Reply:
already merge to one file

---
In reply to: [259967130](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259967130) [](ancestors = 259967130)